README.md

```diff
@@ -395,3 +398,34 @@ The [ss-open/ci/recipes project](https://gitlab.com/ss-open/ci/recipes) offers a
 
 If you have any questions, suggestions, need a wrapper for a programming language or just want to chat join #editorconfig-checker on freenode(IRC).
 If you don't have an IRC-client set up you can use the [freenode webchat](https://webchat.freenode.net/?channels=editorconfig-checker).
+
+## Contributing
+
+Anyone can help to improve the project, submit a Feature Request, a bug report or even correct a spelling mistake.
+
+The steps to contribute can be found in the [CONTRIBUTING.md](./CONTRIBUTING.md) file.
+
+## Semantic Versioning Policy
+
+**editorconfig-checker** adheres to [Semantic Versioning](https://semver.org/) for releases.
+
+However, as it is a code quality tool, it's not always clear when a minor or major version bump occurs. The following rules are used to determine the version bump:
+
+- Patch release (1.0.x -> 1.0.y)
+  - Updates to format outputs (error messages, logs, ...).
+  - Performance improvements which doesn't affect behavior.
+  - Build process changes (e.g., updating dependencies, updating `Dockerfile`, ...).
+  - Reverts (reverting a previous commit).
+  - Bug fixes which result in **editorconfig-checker** reporting less linting errors, also known as "false-positives".
+- Minor release (1.x.0 -> 1.y.0)
+  - Adding new [configuration](./README.md#configuration) options, including new CLI flags.
+  - Adding new [default path](./README.md#default-excludes) to exclude by default.
+  - Adding new [format output](./README.md#output-formats).
+  - Support a new [editorconfig](https://editorconfig.org/) property (e.g: `insert_final_newline`, `indent_size`, ...).
+  - Any new feature which doesn't break existing behavior.
+- Major release (x.0.0 -> y.0.0)
+  - Removal/Rename of a [configuration](./README.md#configuration) option.
```

Comment:
I think we will always follow the pattern of adding a new config option, keeping the old one valid for another release (or two), warning users that this option is deprecated and they should migrate to the new one, before we finally remove the old option. (that should maybe also be an addition to the maintainer guide.)
Hence we do not really need to mention renames, since they do not exist under this model.

```suggestion
  - Removal of a [configuration option](./README.md#configuration).
```